src/libs/actions/PaymentMethods.ts

```diff
@@ -28,45 +46,41 @@ function continueSetup() {
 }
 
 function openWalletPage() {
-    const onyxData = {
-        optimisticData: [
-            {
-                onyxMethod: Onyx.METHOD.MERGE,
-                key: ONYXKEYS.IS_LOADING_PAYMENT_METHODS,
-                value: true,
-            },
-        ],
-        successData: [
-            {
-                onyxMethod: Onyx.METHOD.MERGE,
-                key: ONYXKEYS.IS_LOADING_PAYMENT_METHODS,
-                value: false,
-            },
-        ],
-        failureData: [
-            {
-                onyxMethod: Onyx.METHOD.MERGE,
-                key: ONYXKEYS.IS_LOADING_PAYMENT_METHODS,
-                value: false,
-            },
-        ],
-    };
-
-    return API.read('OpenPaymentsPage', {}, onyxData);
+    return API.read(
+        'OpenPaymentsPage',
+        {},
+        {
+            optimisticData: [
+                {
+                    onyxMethod: Onyx.METHOD.MERGE,
+                    value: true,
+                },
+            ],
+            successData: [
+                {
+                    onyxMethod: Onyx.METHOD.MERGE,
+                    value: false,
+                },
+            ],
+            failureData: [
+                {
+                    onyxMethod: Onyx.METHOD.MERGE,
+                    key: ONYXKEYS.IS_LOADING_PAYMENT_METHODS,
+                    value: false,
+                },
+            ],
+        },
+    );
 }
 
-/**
- *
- * @param {Number} bankAccountID
- * @param {Number} fundID
- * @param {Object} previousPaymentMethod
- * @param {Object} currentPaymentMethod
- * @param {Boolean} isOptimisticData
- * @return {Array}
- *
- */
-function getMakeDefaultPaymentOnyxData(bankAccountID, fundID, previousPaymentMethod, currentPaymentMethod, isOptimisticData = true) {
-    const onyxData = [
+function getMakeDefaultPaymentOnyxData(
+    bankAccountID: number,
+    fundID: number,
+    previousPaymentMethod: PaymentMethod,
+    currentPaymentMethod: PaymentMethod,
+    isOptimisticData = true,
+): DefaultPaymentOnyxDataItem[] {
+    const onyxData: DefaultPaymentOnyxDataItem[] = [
```

Comment:
@JKobrynski I think you can use `OnyxUpdate[]` type instead, so consider if you need `DefaultPaymentOnyxDataItem` at all

Reply:
Updated, thanks!